website/docs/r/monitor.html.markdown

```diff
@@ -400,26 +406,26 @@ Here is a summary of arguments for each condition type (fields which are not mar
 #### logs_static_condition
   - `field`
   - `critical`
-    - `time_range` (Required)
+    - `time_range` (Required) : Accepted format: Optional `-` sign followed by `<number>` followed by a `<time_unit>` character: `s` for seconds, `m` for minutes, `h` for hours, `d` for days. Examples: `30m`, `-12h`.
```

Comment:
currently i think users are leveraging +ve values not -ve. see my comment above

Reply:
I commented above that this is not the case. Here I added a description for the field format, with optional `-` sign.